README.md

```diff
@@ -0,0 +1,179 @@
+## ORGANIZATION: Carleton University
+---
+### *AUTHORS:*  
+---
+### Originally Developed by	: [Dr. Cristina Ruiz Martin](https://github.com/cruizm)
+### Modified by				: [Rajeev Kuamr Jandhyala](https://github.com/jrajeev459) and [Abhinav Achanta](https://github.com/abhinavachanta)
+
+### Table of Contents 
+------
+- [Project Description](#Project_Description)
+- [File Structure](#File_Structure)
+- [Installation Guide](#Installation_Guide)
+- [Steps to Run Simulator](#Steps_to_Run_Simulator)
+
+---
+
+## Project Description
+---
+Alternate Bit Protocol(ABP) is for transmission of data packets from sender to reciver through subnet. In order to achieve reliable
+transmission between sender and receiver through unreliable network the ABP protocol is used. when sender sends the packet,
+it waits for particular to recieve acknowledgement from reviever. if it is not arrived in that time it re-sends packet until it gets
+the expected acknowledgement before sending next packet. In order to distinguish from one packet to another this ABP helps to add
+an additional bit on each packet.
+
+There are three components of ABP simulator are 
+
+**Sender** - Its behavior is pretty complex. When an external signal is received the sender changes from initial phase *passive*
+to *active*. Once activated it start sending the packet with the alternating bit. Every time a packet is sent,it waits for an
+acknowledgement during an specific waiting time. It the acknowledgment does not arrive within the time window, the sender will
+re- send the previous packet with the alternating bit. If the expected acknowledgement is received within the time window, the 
+sender will send the next packet. When there are no more packets to send, the sender will go again to the *passive* phase.
+
+**subnet** - The subnets just pass the packets after a time delay. However, in order to simulate the unreliability of the network,
+only 95% of the packets will be passed in each of the subnet, i.e. 5% of the data will be lost through the subnet
+
+**Receiver** - The behavior of receiver is to receive the data and send back an acknowledgement extracted from the received
+data after a time period.
+
+---
+
+# File Structure
+---
+
+##### bin [This folder containes the .exe files which are generated during program execution]
+1. src [This folder contains the simulator .exe file]
+	- ABP.exe
+2. test [This folder contains the test .exe files]
+	- SUBNET_TEST.exe
+	- SENDER_TEST.exe
+	- RECEIVER_TEST.exe
+		
+##### build [This folder contains .o files which are generated during program execution]
+1. src [This folder contains simulator .o files]
+	- main.o
+	- message.o
+2. test [This folder contains the test .o files]
+	1. subnet
+		- main.o
+	2. sender
+		- main.o
+	3. receiver
+		-main.o
+			
+##### data [This folder contains input data for testing and output files]
+- input_abp_0.txt
+- input_abp_1.txt
+- abp_output.txt
+	
+##### doc [This folder contains documents related to project]
```

Comment:
-alternatebitprot.pdf -Cadmium_Documentation_Ubuntu.pdf -Cadmium_Windows.pdf needs spacing
such as - alternatebitprot.pdf  and so on